Main/Service.java

```diff
@@ -0,0 +1,620 @@
+package Main;

```

Comment:
I would recommend to split this service in multiple services - OrderService, ProductService, RestaurantService, UserService, etc - depending on how you thought the methods, each of the methods in its specific service. In main you can have the chooseOptions methods in which you can call the methods from your service.

Reply:
this way is easier to read, your code is more decoupled which will help you a lot in time when you will add new functionality/refactor your code